# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../yaboot/yaboot_conf_example.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

don't replace the real /etc/yaboot.conf, but install a .example file for reference

--- yaboot-1.3.13/Makefile.orig
+++ yaboot-1.3.13/Makefile
@@ -201,12 +201,7 @@
 	install -o root -g root -m 0644 man/ybin.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/ybin.8.gz
 	install -o root -g root -m 0644 man/yaboot.conf.5.gz ${ROOT}/${PREFIX}/${MANDIR}/man5/yaboot.conf.5.gz
 	@gunzip man/*.gz
-	@if [ ! -e ${ROOT}/etc/yaboot.conf ] ; then						\
 		echo "install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf"; \
-		install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf;	\
+		install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf.example;	\
-	 else											\
-		echo "/etc/yaboot.conf already exists, leaving it alone";			\
-	 fi
 	@echo
 	@echo "Installation successful."
 	@echo
